www/src/views/starter-library/filtered-starters.js

```diff
@@ -84,8 +84,6 @@ export default class FilteredStarterLibrary extends Component {
     if (urlState.s.length > 0) {
       starterNodes = starterNodes.filter(starter =>
         JSON.stringify(starter.node)
-          .toLowerCase()
-          .includes(urlState.s.toLowerCase())
```

Comment:
But you are removing the filter logic here, aren't you?

Reply:
Yes @rathesDot , you are right I `rm` these `lines`, Its must be filtered, what you think is it the better ...